webapp/src/main/webapp/src/app/sender/archive/sender-archive.component.html

```diff
@@ -0,0 +1,77 @@
+<!--View sync message overall status-->
+<div>
+	<div class="row">
+		<div class="col-md-8">
+		</div>
+		<div class="col-md-4">
+			<div>
+				<table style="left: 0;">
+					<tr>
+						<td></td>
+						<td colspan="4"></td>
+						<td colspan="2" i18n="@@common-start">Start Date</td>
+						<td></td>
+						<td i18n="@@common-end">End Date </td>
+						<td> </td>
+					</tr>
+					<tr>
+						<td >
+							<button class="btn btn-outline-primary" (click)="searchByPeriod($event)">
+								<span i18n="@@common-search-period">Search by Period</span>
+							</button>
+						</td>
+						<td colspan="4"></td>
+						<td colspan="2">
+							<input type="date" [(ngModel)]="searchEvent.startDate" />
+						</td>
+						<td></td>
+						<td>
+							<input type="date" [(ngModel)]="searchEvent.endDate" />
+						</td>
+					</tr>
+				</table>
+			</div>
+		</div>
+	</div>
+	<!--Archived message -->
+	<div>
+		<h5><span class="badge badge-dark" i18n="@@common-total-count">Total Count</span>&nbsp;<span
+				class="badge border border-secondary">{{count}}</span></h5>
+		<div class="ui-bar-spacer">
+			<table datatable [dtOptions]="dtOptions" [dtTrigger]="dtTrigger" class="table table-striped text-center">
+				<thead>
+					<tr>
+						<th i18n="@@common-table-name">Table Name</th>
+						<th i18n="@@common-identifier">Identifier</th>
+						<th i18n="@@common-operation">Operation</th>
+						<th i18n="@@common-dateAdded">Date Created</th>
```

Comment:
I think date created is not important to display for a user, you can omit in the UI

Reply:
Done